lib/workers/pr/changelog/release-notes.ts

```diff
@@ -179,14 +180,17 @@ export async function getReleaseNotesMd(
     if (changelogParsed.length >= 2) {
       for (const section of changelogParsed) {
         try {
-          const [heading] = section.split('\n');
+          // replace brackets and parenthesis with space
+          const deParenthesizedSection = section.replace(/[[\]()]/g, ' ');
+          const [heading] = deParenthesizedSection.split('\n');
           const title = heading
             .replace(/^\s*#*\s*/, '')
             .split(' ')
             .filter(Boolean);
           let body = section.replace(/.*?\n(-{3,}\n)?/, '').trim();
           for (const word of title) {
-            if (word.includes(version)) {
+            // const isUrl = /^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-z0-9]+([-.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$/gm;
+            if (word.includes(version) && !URL.parse(word).hostname) {
```

Comment:
`URL.parse()` can throw - see https://nodejs.org/docs/latest-v12.x/api/url.html#url_url_parse_urlstring_parsequerystring_slashesdenotehost

So please make a function `isUrl` that includes try/catch to be safest

Reply:
sure